src/components/extensive/Modal/ModalWithMap.tsx

```diff
@@ -48,7 +48,7 @@ const ModalWithMap: FC<ModalWithMapProps> = ({
             <Icon name={IconNames.WRI_LOGO} width={108} height={30} className="min-w-[108px]" />
             <div className="flex items-center">
               <When condition={status}>
-                <Status status={status ?? "draft"} />
+                <Status status={(status ? status : "draft") as StatusEnum} />
```

Comment:
Please revert to `status ?? 'draft'`

Reply:
Thanks for the feedback! I've reverted the code to use status ?? 'draft' for better readability and clarity.